src/lib/translators/mongo.ts

```diff
@@ -410,28 +410,31 @@ function getOperator(op: string) {
 /**
  * Translate a mathjs logical tree describing a formula into a Mongo step
  * @param node a mathjs node object (usually received after parsing an string expression)
+ * @param pseudos an object which keys are pseudo and values are corresponding column name
  * This node is the root node of the logical tree describing the formula
  */
-function buildMongoFormulaTree(node: MathNode): MongoStep | string | number {
-  // For type checking in `case: 'OperatorNode'` in the`switch`clause below,
-  // do not let`args` and `op` be potentially`undefined`
+function buildMongoFormulaTree(
+  node: MathNode,
+  pseudos: { [key: string]: string },
```

Comment:
It's a matter of style but I'm getting used to `Record<string, string>` and now I think it's a bit clearer. What do you think?